cms/envs/common.py

```diff
@@ -1136,3 +1136,6 @@
 
 # OpenID Connect issuer ID. Normally the URL of the authentication endpoint.
 OAUTH_OIDC_ISSUER = 'https://www.example.com/oauth2'
+
+# URL which is used to convert math formula to Content MathML in Symbolic Response problem.
+SYMMATH_URL = 'https://math-xserver.mitx.mit.edu/snuggletex-webapp-1.2.2/ASCIIMathMLUpConversionDemo'
```

Comment:
the idea is to move this into a configuration file and not have it be in the public repo at all.